.github/workflows/upgrade_path_testing.yaml

```diff
@@ -146,8 +195,215 @@ jobs:
+          nohup zetavisor start --rpc.laddr tcp://0.0.0.0:26657 --minimum-gas-prices ${GAS_PRICES} "--grpc.enable=true" > cosmovisor.log 2>&1 &
+
+          sleep ${UPGRADES_SLEEP_TIME}
+          cat cosmovisor.log
+          echo "*********CHECK VERSION AFTER BINARY SWITCH*********"
+          check_version=$(zetavisor version | tr -d '\n')
+          http_version=$(curl http://127.0.0.1:26657/abci_info | jq .result.response.version -r | tr -d '\n')
+          
+          echo "END_VERSION=${{ github.event.inputs.version }}"
+          echo "CURRENT_VERSION_BINARY=${check_version}"
+          echo "CURRENT_VERSION_HTTP=${http_version}"
+          
+          if [ "${{ github.event.inputs.version }}" == "${check_version}" ]; then
+            if [ "${{ github.event.inputs.version }}" == "${http_version}" ]; then
+              echo "*********VERSION MATCHES UPGRADE SUCCESS*********"
+              exit 0
+            else
+              echo "*********VERSION DOESN'T MATCH UPGRADE FAILED*********"
+              exit 2
+            fi
+          else
+            echo "*********VERSION DOESN'T MATCH UPGRADE FAILED*********"
+            exit 2
+          fi
+
+      - name: "CONCENSUS_BREAKING_UPGRADE"
+        if: env.UPGRADE_TYPE == 'GOV' #GOV
+        run: |
+          echo "*****UPGRADE TYPE*****"
+          echo ${UPGRADE_TYPE}
+          
+          echo "*****BUILD GOV PROPOSAL*****"
+          GOV_PROPOSAL=$(python .github/actions/upgrade-testing/scripts/raise_gov_proposal.py)
+          echo "${GOV_PROPOSAL}"
+          
+          GOV_PROPOSAL_OUTPUT=$(eval ${GOV_PROPOSAL})
+          
+          echo "*****GOV PROPOSAL OUTPUT*****"
+          echo ${GOV_PROPOSAL_OUTPUT}
+          
+          echo "*****GET TX HASH*****"
+          TX_HASH=$(echo ${GOV_PROPOSAL_OUTPUT} | awk -F'txhash: ' '{print $2}' | tr -d '\n' | tr -d ' ')
+          echo "****TXHASH: ${TX_HASH}****"
+          
+          echo "*****SLEEP FOR 1 MIN TO ALLOW TX TO MAKE IT ON NETWORK*****"
+          sleep 15
+
+          zetacored query tx --type=hash ${TX_HASH}
+          
+          proposal_id=$(python .github/actions/upgrade-testing/scripts/get_proposal_id.py)
+          echo "****PROPOSAL_ID: ${proposal_id}****"
+          
+          source ${GITHUB_ENV}
+          
+          zetacored tx gov vote "${proposal_id}" yes \
+            --from ${MONIKER} \
+            --keyring-backend test \
+            --chain-id ${CHAINID} \
+            --node http://127.0.0.1:26657 \
+            --gas=auto \
+            --gas-adjustment=2 \
+            --gas-prices=${{ env.GAS_PRICES }} \
+            -y
+          
+          sleep 5
+          zetacored query gov proposal ${proposal_id} --node http://127.0.0.1:26657
+          
+          TARGET_HEIGHT=$(echo ${UPGRADE_HEIGHT} | cut -d '.' -f 1)
+          ENDPOINT="http://127.0.0.1:26657"
+          MAX_TRIES=100
```

Comment:
Can you set these and env variables at the job level? Just so it's easier to see what values can be tweaked for fine tuning